src/QuiqChatClient.js

```diff
@@ -30,7 +30,7 @@ class QuiqChatClient {
   initialized: boolean;
   clientInactiveTimer: number;
 
-  constructor(host: string, contactPoint: string) {
+  initialize = (host: string, contactPoint: string) => {
```

Comment:
👍 I'm guessing this will also change the way Webchat consumes this.  Also, could you update the README to reflect this change?

Reply:
Yep that was the plan after I made sure this was the right direction on this story. I have updated the README as well.